scanpipe/pipes/rootfs.py

```diff
@@ -265,6 +265,8 @@ def scan_rootfs_for_system_packages(project, rootfs):
     installed_packages = rootfs.get_installed_packages(package_getter)
     for index, (purl, package) in enumerate(installed_packages):
         logger.info(f"Creating package #{index}: {purl}")
+        if package.namespace != distro_id:
```

Comment:
This is related to: 

In SCTK we are now getting the namespace populated from clues present in package data with this [commit](https://github.com/nexB/scancode-toolkit/pull/3647/commits/1cd5d02b3ecf30c3dcfaedc41e1d9d1a9802240c) and if we don't have any clues we are assigning a defualt value to the namespace which is `debian` for debian packages.

In SCIO we get distro data from os-release which overrides these values, as they are always correct and takes care of occasional false-positives and default debian cases.

Reply:
Per https://github.com/GoogleContainerTools/distroless 

see https://github.com/GoogleContainerTools/distroless/blob/f55b2b343481f52ef3bde34c8a2f3631a40a36c1/debian_packages.yaml#L75 ... the namespace is debian in all cases.... some deb11 some deb12. Eventually the way to go will be to match these.

Reply:
In that case we are good, thanks!